src/android/CameraManager.java

```diff
@@ -108,6 +111,14 @@ public void openDriver() throws IOException {
 			}
 			
 			if (DEBUG) Log.i(TAG, "Camera open success");
+
+			// Get the camera profile. Low quality is adequate for display our demo purposes right now.
+			camcorderProfile = CamcorderProfile.get(CamcorderProfile.QUALITY_LOW);
+
+			param = camera.getParameters();
+			param.setPreviewFrameRate(camcorderProfile.videoFrameRate);
+			camera.setParameters(param);
```

Comment:
What would you think of cycling through the CamcorderProfiles, beginning with QUALITY_<whatever is highest> and working downwards, until we find one that doesn't throw an error?